rancher-vcluster/rancher-vcluster.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: harvesterhci.io/v1beta1
+kind: Addon
+metadata:
+  name: rancher-vcluster
+  namespace: harvester-system
+  labels:
+    addon.harvesterhci.io/experimental: "true"
+spec:
+  enabled: false
+  repo: https://charts.loft.sh
+  version: "v0.15.2"
+  chart: vcluster
+  valuesContent: |-
+    hostname: "rancher.172.19.108.3.sslip.io"
```

Comment:
@ibrokethecloud  
This is awesome :smile:  - I was wondering, would it be useful to default it from:
- *.sslip.io
To something like:
- *.nip.io

If a *.nip.io was used for the `hostname` the user (as long as this wasn't airgapped) - wouldn't need to modify:
- /etc/hosts
- adding a custom dns entry somewhere

https://nip.io/

Maybe it's not worth the edit :sweat_smile:  - just wanted to mention it as a possibility for a default value